shared/src/commonMain/kotlin/org/hyperskill/app/track_list/domain/analytic/TrackListTrackModalHiddenHyperskillAnalyticEvent.kt

```diff
@@ -0,0 +1,40 @@
+package org.hyperskill.app.track_list.domain.analytic
+
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticAction
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticEvent
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticPart
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticRoute
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticTarget
+
+/**
+ * Represents a hidden analytic event of the new user placeholder track modal.
+ *
+ * JSON payload:
+ * ```
+ * {
+ *     "route": "/tracks",
+ *     "action": "hidden",
+ *     "part": "track_modal",
+ *     "target": "close",
+ *     "context":
+ *     {
+ *         "track_id": 1234
+ *     }
+ * }
+ * ```
+ * @see HyperskillAnalyticEvent
+ */
+class TrackListTrackModalHiddenHyperskillAnalyticEvent(val trackId: Long) : HyperskillAnalyticEvent(
+    HyperskillAnalyticRoute.Tracks(),
+    HyperskillAnalyticAction.HIDDEN,
+    HyperskillAnalyticPart.TRACK_MODAL,
+    HyperskillAnalyticTarget.CLOSE
+) {
+    companion object {
+        private const val PARAM_TRACK_ID = "track_id"
+    }
+
+    override val params: Map<String, Any>
+        get() = super.params +
+            mapOf(PARAM_CONTEXT to mapOf(PARAM_TRACK_ID to trackId))
+}
```

Comment:
We can do the same as we do for projects
<img width="697" alt="Screenshot 2023-05-12 at 4 10 24 PM" src="https://github.com/hyperskill/mobile-app/assets/8586063/069c7be4-4ac2-43bf-a1ab-13429e6e9b85">

Reply:
@vladkash 
I added `Tracks` AnalytsRoute in ProjectsSelectionList branch.
https://github.com/hyperskill/mobile-app/blob/27e9d22d5740da58418c767f6a697be436b7799c/shared/src/commonMain/kotlin/org/hyperskill/app/analytic/domain/model/hyperskill/HyperskillAnalyticRoute.kt
I'll merge it into develop and you can use it in your branch.

Reply:
@XanderZhu aded Analytics route contains `trackId` parameter, but when we are on the track list screen, there is not specific track right now, it'll appear after track selection. Therefore, for tracks list screen we need route `/tracks` instead of `/tracks/{id}`. I made changes to your route.